enterprise/frontend/src/metabase-enterprise/table-actions/execution/TableActionExecuteModalContent.tsx

```diff
@@ -49,21 +55,33 @@ export const TableActionExecuteModalContent = ({
     }
   }, [action, actionOverrides]);
 
+  const [executeAction] = useExecuteActionMutation();
+
   const handleSubmit = useCallback(
-    (parameters: ParametersForActionExecution) => {
+    async (parameters: ParametersForActionExecution) => {
       const fullParameters = {
         ...initialValues,
         ...parameters,
       };
 
-      return dispatch(
-        executeAction({
-          action: checkNotNull(action),
-          parameters: fullParameters,
-        }),
-      );
+      const result = await executeAction({
+        actionId: actionId as number,
+        parameters: fullParameters,
```

Comment:
What do you think about renaming this key from `parameters` to `input` so its consistent with the API?

Reply:
The thing is, it's called "action parameters" all across the app, that's why I've decided to leave the name as is. I think it gives more context, on what we're actually executing – action with parameters.